modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationModelIndexer.java

```diff
@@ -585,6 +590,8 @@ private synchronized void _stopBundleTracker() {
 	private ConfigurationModelIndexer.
 		ConfigurationModelsClusterMasterTokenTransitionListener
 			_configurationModelsClusterMasterTokenTransitionListener;
+	private ServiceRegistration<IdentifiableOSGiService>
+		_destinationServiceRegistration;
```

Comment:
Hey @Valmir-unicap, rename it to `ServiceRegistration<IdentifiableOSGiService>`. Thx